bambi/plots/__init__.py

```diff
@@ -1,3 +1,16 @@
-from .plot_cap import create_cap_data, plot_cap
+from .create_data import create_cap_data, create_comparisons_data
```

Comment:
Here I would do `from bambi.plots.create_data import ...` always writing `bambi.plots` before `.create_data` so it's not relative anymore.